packages/trader/src/Modules/CFD/Containers/cfd-password-modal.tsx

```diff
@@ -833,33 +968,11 @@ const CFDPasswordModal = ({
     );
 };
 
-CFDPasswordModal.propTypes = {
-    account_title: PropTypes.string,
-    account_type: PropTypes.object,
-    disableCFDPasswordModal: PropTypes.func,
-    email: PropTypes.string,
-    error_message: PropTypes.string,
-    has_cfd_error: PropTypes.bool,
-    is_eu: PropTypes.bool,
-    is_eu_country: PropTypes.bool,
-    is_logged_in: PropTypes.bool,
-    is_cfd_password_modal_enabled: PropTypes.bool,
-    is_cfd_success_dialog_enabled: PropTypes.bool,
-    is_dxtrade_allowed: PropTypes.bool,
-    platform: PropTypes.string,
-    setMt5Error: PropTypes.func,
-    setCFDSuccessDialog: PropTypes.func,
-    submitMt5Password: PropTypes.func,
-    submitCFDPassword: PropTypes.func,
-    cfd_new_account: PropTypes.object,
-};
-
-export default connect(({ client, modules }) => ({
+export default connect(({ client, modules }: RootStore) => ({
     email: client.email,
     account_title: modules.cfd.account_title,
     account_type: modules.cfd.account_type,
     account_status: client.account_status,
-    clearCFDError: modules.cfd.clearCFDError,
```

Comment:
I removed clearCFDError from here since this callback is never used in this module.